creds/src/rangeproof.rs

```diff
@@ -511,7 +511,7 @@ mod tests {
         assert!(Groth16::<CrescentPairing>::verify_with_processed_vk(&pvk, &inputs, &proof).unwrap());
     
         let mut client_state =
-            ClientState::<CrescentPairing>::new(inputs.clone(), proof.clone(), vk.clone(), pvk.clone());
+            ClientState::<CrescentPairing>::new(inputs.clone(), None, proof.clone(), vk.clone(), pvk.clone(), "empty config.json".to_string());
```

Comment:
If we need to specify this, would we use "empty config.json"? Sounds implementation specific. Use something more generic instead, e.g., "empty config"?

Reply:
(changed, as above)